DataCollector/RulerDataCollector.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+
+use Hoathis\Bundle\RulerBundle\Ruler\LoggedRuler;
```

Comment:
@K-Phoen: I let @jubianchi and you decide about the naming. I guess that the Composer package name must at least contain “Symfony” (because “bundle” is too generic). For the code, never mind, go simple. It's up to you to decide.